rfhub2/cli/cli.py

```diff
@@ -1,17 +1,35 @@
-from typing import Tuple
 import click
-from rfhub2.cli.population import LibraryPopulation
+from typing import Tuple
+
+from rfhub2.cli.api_client import Client
+from rfhub2.cli.rfhub_importer import RfhubImporter
 
 
 @click.command()
-@click.argument('paths', nargs=-1, type=click.Path(exists=True))
-@click.option('--no-installed-keywords', 'no_installed_keywords', default=False,
-              help='do not load some common installed keyword libraries, such as BuiltIn.')
-def main(paths: Tuple[str, ...], no_installed_keywords: bool) -> None:
-    """Package to populate rfhub2 with libraries and resource documentation"""
+@click.option('-i', '--app-interface', type=click.STRING, default="http://0.0.0.0",
```

Comment:
I think that for client we should join interface and port in one option like url. This division had more sense when cli was also used for running api itself. Now it seems pointless. Also, 0.0.0.0 should be changed to localhost, it is more sensible from client perspective